packages/edge-login-ui-rn/src/common/actions/LoginAction.js

```diff
@@ -154,13 +142,7 @@ export function userLoginWithPin (data: Object, backupKey?: string) {
   return (dispatch: Dispatch, getState: GetState, imports: Imports) => {
     const { callback, context, folder } = imports
     const myAccountOptions = {
-      ...imports.accountOptions,
-      callbacks: {
-        ...imports.accountOptions.callbacks,
-        onLoggedOut: () => {
```

Comment:
@jamesrisberg Make sure this onLogout callback is being handled

Reply:
Exactly, it doesn't seem to actually break, but I assume it needs to be dispatched. Didn't notice I ripped it out, so I'll need to add it back in appropriately

Reply:
Handling it Friday morning